### PR TITLE
Add task priorities and possibility to remove

### PR DESCRIPTION
**Pull request by dgrochowski**

### CODE REVIEW

File: taskmanager/features/task.feature [hunk 1]
```diff
@@ -26,3 +26,13 @@ Feature: Task management
     Given there is task named "Add switch language button" assigned to user named "Carl"
     When I assigned task named "Add switch language button" to user named "Sarah"
     Then task named "Add switch language button" should be assigned to user named "Sarah"
+
```
Comment (dominikhajduk) [inline taskmanager/features/task.feature:29]: can we add (can be as separate PR) feature related to default priority? That if i do not choose priority `Major` will be set?

**Review by dominikhajduk — APPROVED** [review]

**Review by tarnawski — APPROVED** [review]
